### DISCUSSION TITLE
Transfer Maintainership to community .

### DISCUSSION THREAD

**Discussion by v3ss0n** [AppImage/AppImageKit#1265] [Ideas]
Since this project have almost no maintenance , and there are breaking bugs due to libfuse , AppImage now become unusable to many. I advice maintenance should be transfer to active community members.

**Comment by probonopd** [comment]
> Since this project have almost no maintenance 

This reflects, imho, the ideal state of software: Not constantly changing anymore.

> there are breaking bugs due to libfuse

All that needs to be done is to make AppImageKit use https://github.com/AppImage/type2-runtime.
The hard work has been done.

In my https://github.com/probonopd/go-appimage project I have been using a static runtime for a long time.

> I advice maintenance should be transfer to active community members.

Volunteers?
